kvstore/ikvstore.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2021-2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package kvstore
+
+type IKVStore interface {
+	Init(cfg Config) error
+	Close() error
+	Get(key string) ([]string, error)
+	Set(key string, val string) error
```

Comment:
Since now for a key, multiple values can exist, should the interface for Set should also reflect the same?

Reply:
You are right, this is another thing I want to change in kvstore (the other was mentioned in [this comment](https://github.com/veraison/veraison/pull/134#discussion_r838471280).  In fact, this should be `Append`, not `Set`.

Reply:
See https://github.com/veraison/veraison/issues/135